modin/experimental/engines/omnisci_on_ray/frame/omnisci_worker.py

```diff
@@ -18,11 +18,13 @@
 import pyarrow as pa
 import numpy as np
 
-prev = sys.getdlopenflags()
-sys.setdlopenflags(1 | 256)  # RTLD_LAZY+RTLD_GLOBAL
+if platform == "linux":
```

Comment:
```suggestion
if sys.platform == "linux":
```
otherwise this won't work

Reply:
Done